src/utils/Utils.ts

```diff
@@ -65,6 +66,19 @@ export default class Utils {
     });
   }
 
+  public static getEponymousAttributeValue(resourceInstance: ResourceInstance): string {
+    if (!resourceInstance.resourceType.eponymousAttribute) {
+      return `${resourceInstance.id}`;
```

Comment:
Why is it a template string?

Reply:
it was because the id might be undefined, but I changed it to a guard clause, which solves the problem too.